app/components/modals/invite-new-user.js

```diff
@@ -4,6 +4,7 @@ import {A as emberA} from 'ember-array/utils';
 import run from 'ember-runloop';
 import ModalComponent from 'ghost-admin/components/modals/base';
 import ValidationEngine from 'ghost-admin/mixins/validation-engine';
+import run, {scheduleOnce} from 'ember-runloop';
```

Comment:
oops, should have mentioned it in the other note...this line is partially a duplicate of line 4 😄